Add tables to utility page (#552)
matter-labs/zksync-web-era-docs: docs/api/js/utils.md

#### Inputs

| Parameter | Type          | Description     |
| --------- | ------------- | --------------- |

> | `address` | string        | Token address.  |

Reviewer: this is not a token address

#### Inputs

| Parameter      | Type                  | Description      |
| -------------- | --------------------- | ---------------- |
| `sender`       | string                | Senders address. |

> | `senderNonce`  | `BigNumberish` object | Senders nonce.   |

Reviewer: if you're going to use the possessive, do it correctly

#### Inputs

| Parameter           | Type                             | Description                                                  |
| ------------------- | -------------------------------  | ------------------------------------------------------------ |

> | `providerL2`        | object                           | zkSync provider.                                             |

Reviewer: ```suggestion
| `providerL2`        | `Provider` object                           | zkSync provider.                                             |
```

| `l1BridgeAddress`   | string                           | L1 bridge address.                                              |
| `l2BridgeAddress`   | string                           | L2 bridge address.                                              |
| `token`             | string                           | Token address.                                               |
| `amount`            | `BigNumberish` object            | Deposit amount.                                              |
| `to`                | string                           | Recipient address.                                           |

> | `bridgeData`        | `BytesLike` object               | Data.                                                        |

Reviewer: ```suggestion
| `bridgeData`        | `BytesLike` object               | Bridge data.                                                        |
```

> | Parameter           | Type                             | Description                                                  |
> | ------------------- | -------------------------------  | ------------------------------------------------------------ |
> | `providerL2`        | object                           | zkSync Provider.                                             |
> | `l1BridgeAddress`   | string                           | Bridge Address.                                              |
> | `l2BridgeAddress`   | string                           | Bridge Address.                                              |
> | `token`             | string                           | Token Address.                                               |
> | `amount`            | `BigNumberish` object            | Deposit amount.                                              |
> | `to`                | string                           | Recipient Address.                                           |
> | `bridgeData`        | `BytesLike` object               | Data.                                                        |
> | `from?`             | string (optional)                | Sender Address.                                              |
> | `gasPerPubdataByte?`| `BigNumberish` object (optional) | Current gas per byte of pubdata.                             |

Reviewer: ```suggestion
| Parameter           | Type                             | Description                                                  |
| ------------------- | -------------------------------  | ------------------------------------------------------------ |
| `providerL2`        | object                           | zkSync provider.                                             |
| `l1BridgeAddress`   | string                           | L1 bridge address.                                              |
| `l2BridgeAddress`   | string                           | L2 bridge address.                                              |
| `token`             | string                           | Token address.                                               |
| `amount`            | `BigNumberish` object            | Deposit amount.                                              |
| `to`                | string                           | Recipient address.                                           |
| `bridgeData`        | `BytesLike` object               | Data.                                                        |
| `from?`             | string (optional)                | Sender address.                                              |
| `gasPerPubdataByte?`| `BigNumberish` object (optional) | Current gas per byte of pubdata.                             |
```

#### Inputs

> | Parameter           | Type                             | Description                                   |
> | ------------------- | -------------------------------- | --------------------------------------------- |
> | `providerL1`        | object                           | ethers Provider.                              |
> | `providerL2`        | string                           | zkSync Provider.                              |
> | `token`             | string                           | Token Address.                                |
> | `amount`            | `BigNumberish` object            | Deposit amount.                               |
> | `to`                | string                           | Recipient Address.                            |
> | `from?`             | string (optional)                | Sender Address.                               |
> | `gasPerPubdataByte?`| `BigNumberish` object  (optional)| Current gas per byte of pubdata.              |

Reviewer: ```suggestion
| Parameter           | Type                             | Description                                   |
| ------------------- | -------------------------------- | --------------------------------------------- |
| `providerL1`        | Provider object                           | Ethers provider.                              |
| `providerL2`        | Provider object                         | zkSync provider.                              |
| `token`             | string                           | Token address.                                |
| `amount`            | `BigNumberish` object            | Deposit amount.                               |
| `to`                | string                           | Recipient address.                            |
| `from?`             | string (optional)                | Sender address.                               |
| `gasPerPubdataByte?`| `BigNumberish` object  (optional)| Current gas per byte of pubdata.              |
```

#### Inputs

| Parameter           | Type                         | Description                                                  |
| ------------------- | ---------------------------  | ------------------------------------------------------------ |

> | `receipt`           | object                       | Transaction receipt.                                         |

Reviewer: ```suggestion
| `receipt`           | TransactionReceipt object                     | Transaction receipt.                                         |
```

> | `receipt`           | TransactionReceipt object                     | Transaction receipt.                                         |

Reviewer: ```suggestion
| `receipt`           | `TransactionReceipt` object                     | Transaction receipt.                                         |
```

#### Inputs

| Parameter           | Type                         | Description                                   |
| ------------------- | ---------------------------  | --------------------------------------------- |
| `l1TokenAddress`    | string                       | Token address on L1.                          |
| `l1Sender`          | string                       | Sender address on L1.                         |
| `l2Receiver`        | string                       | Recipient address on L2.                      |
| `amount`            | `BigNumberish` object        | Gas fee for the number of tokens to bridge.   |

> | `bridgeData`        | `BytesLike` object           | Data .                                        |

Reviewer: ```suggestion
| `bridgeData`        | `BytesLike` object           | Data                                        |
```

#### Inputs

| Parameter           | Type                             | Description                                   |
| ------------------- | -------------------------------- | --------------------------------------------- |
| `l1TokenAddress`    | string                           | Token address on L1.                          |

> | `provider`          | object                           | Ethers Provider.                              |

Reviewer: ```suggestion
| `provider`          | Provider object                           | Ethers provider.                              |
```

#### Inputs

| Parameter           | Type                        | Description                          |
| ------------------- | --------------------------- | ------------------------------------ |
| `txReceipt`         | `TransactionReceipt` object | Receipt of the L1 transaction.       |

> | `zkSyncAddress`     | `Address` string            | Address of zkSync Era main contract. |

Reviewer: ```suggestion
| `zkSyncAddress`     | `Address` as string            | Address of zkSync Era main contract. |
```

#### Inputs

| Parameter           | Type                | Description                           |
| ------------------- | ------------------- | ------------------------------------- |

> | `sender`            | `Address` string    | The sender of the message on L2.      |

Reviewer: ```suggestion
| `sender`            | `Address` as string    | The sender of the message on L2.      |
```

#### Inputs

| Parameter           | Type                    | Description                  |
| ------------------- | ----------------------- | ---------------------------  |

> | `provider`          | `Provider` object       | The RPC provider.            |

Reviewer: ```suggestion
| `provider`          | `Provider` object       | Provider.            |
```

> | `address`           | string                  | Sender address as string.    |

Reviewer: ```suggestion
| `address`           | string                  | Sender address.    |
```

#### Inputs

| Parameter           | Type                    | Description                  |
| ------------------- | ----------------------- | ---------------------------  |

> | `token`             | `Address` object        | The token adress.            |

Reviewer: ```suggestion
| `token`             | `Address` object        | The token address.            |
```

#### Inputs

| Parameter           | Type                    | Description                  |
| ------------------- | ----------------------- | ---------------------------  |

> | `provider`          | `Provider` object       | The RPC provider.            |

Reviewer: ```suggestion
| `provider`          | `Provider` object       | Provider.            |
```

#### Inputs

| Parameter           | Type                    | Description                  |
| ------------------- | ----------------------- | ---------------------------  |

> | `provider`          | `Provider` object       | The RPC provider.            |
> | `address`           | string                  | Sender address as string.    |
> | `msgHash`           | string                  | The hash of the message.     |
> | `signature`         | `SignatureLike` object  | Ethers signature.            |

Reviewer: ```suggestion
| `provider`          | `Provider` object       | Provider.            |
| `address`           | string                  | Sender address.    |
| `msgHash`           | string                  | The hash of the message.     |
| `signature`         | `SignatureLike` object  | Ethers signature.            |
```

#### Inputs

| Parameter           | Type                       | Description                                                 |
| ------------------- | -------------------------- | ----------------------------------------------------------  |
| `provider`          | `Provider` object          | Provider.                                           |
| `address`           | string                     | Sender address.                                   |
| `domain`            | `TypedDataDomain` object   |       Domain data.                                                      |

> | `types`             | Map<string, TypedDataField>    | Map of records pointing from field name to field type.  |

Reviewer: ```suggestion
| `types`             | `Map<string, TypedDataField>`    | Map of records pointing from field name to field type.  |
```

> | `value`             | Record<string, any>                     | A single record value.                                           |

Reviewer: ```suggestion
| `value`             | `Record<string, any>`                     | A single record value.                                           |
```

> | `provider`          | `Provider` object          | The RPC provider.                                           |
> | `address`           | string                     | Sender address as string.                                   |
> | `domain`            | `TypedDataDomain` object   |                                                             |
> | `types`             | string                     | The map of records pointing from field name to field type.  |
> | `value`             | string                     | A single record.                                            |
> | `signature`         | `SignatureLike` object     | Ethers signature.                                           |

Reviewer: ```suggestion
| `provider`          | `Provider` object          | Provider.                                           |
| `address`           | string                     | Sender address.                                   |
| `domain`            | `TypedDataDomain` object   |       Domain data.                                                      |
| `types`             | Map<string, TypedDataField>    | Map of records pointing from field name to field type.  |
| `value`             | Record<string, any>                     | A single record value.                                           |
| `signature`         | `SignatureLike` object     | Ethers signature.                                           |
```

#### Inputs

| Parameter           | Type        | Description                                                        |
| ------------------- | ----------- | -----------------------------------------------------------------  |

> | `millis`            | number      | The number of time required in milliseconds to pause an execution. |

Reviewer: ```suggestion
| `millis`            | number      | Number of milliseconds. |
```
